runtime/compiler/env/J9ClassEnv.cpp

```diff
@@ -24,11 +24,14 @@
 #include "env/ClassEnv.hpp"
 #include "env/CompilerEnv.hpp"
 #include "env/jittypes.h"
+#include "env/StackMemoryRegion.hpp"
```

Comment:
If you don't need the StackMemoryRegion please remove this header addition as well.